examples/SLAM/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "slam-camera-example",
+  "version": "1.0.0",
+  "description": "SLAM camera demo. Runs using webAPI with fallback to 8thwall",
+  "main": "js/index.js",
+  "type": "module",
+  "module": "js/index.js",
+  "scripts": {
+    "build": "esbuild ./js/index.js --bundle --outfile=\"deploy/SLAM-camera-example-bundle.js\""
+  },
+  "keywords": [
+    "wonderland-engine"
+  ],
+  "devDependencies": {
+    "esbuild": "^0.15.18",
+    "typescript": "^4.9.5"
+  },
+  "dependencies": {
+    "@wonderlandengine/api": "^0.9.8",
+    "@wonderlandengine/components": "^0.9.2",
```

Comment:
If you use `1.0.0-rc.2` here, you get the new class based components! 🎉

Reply:
Actually, I don't use them anywhere so far. Removing.